Parser/asdl_c.py

```diff
@@ -1065,16 +1065,22 @@ def visitModule(self, mod):
     }
 
     PyObject *dict = NULL, *fields = NULL, *remaining_fields = NULL,
-             *remaining_dict = NULL, *positional_args = NULL;
+             *positional_args = NULL;
```

Comment:
```suggestion
    PyObject *dict = NULL, *fields = NULL, *positional_args = NULL;
```

I don't think you need the `remaining_fields` (but I may have skipped it).

Reply:
Thanks! Removed it.